### ISSUE TITLE
Invalid values in KL divergence computation of VAT

### ISSUE THREAD

**Issue by ririnicolae** [IBM/adversarial-robustness-toolbox#120]
**Describe the bug**
This issue was discovered during the development for #75. Because the model predictions / logits can be negative, the computation of the KL divergence in virtual adversarial method can yield invalid values (`nan` and `inf`), preventing the method from producing meaningful results.

**System information (please complete the following information):**
 - OS: Ubuntu
 - Python version: 3.6
 - ART version or commit number: 3d618e3
